staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -257,8 +257,8 @@ type VirtualMachineInstanceStatus struct {
 	// +optional
 	KernelBootStatus *KernelBootStatus `json:"kernelBootStatus,omitempty"`
 
-	// FSFreezeStatus is the state of the fs of the guest
-	// it can be either frozen or thawed
+	// FSFreezeStatus reflects whether a freeze or thaw command was executed,
+	// not the actual filesystem state.
```

Comment:
shouldn't this be the same comment as the other?

Reply:
Yep! thanks for catching this.